src/worker.cc

```diff
@@ -471,9 +471,16 @@ void from_json(rapidjson::Document& doc, odin::DirectionsOptions& options) {
     options.set_language(*language);
   }
 
+  // deprecated
   auto narrative = rapidjson::get_optional<bool>(doc, "/narrative");
-  if (narrative) {
-    options.set_narrative(*narrative);
+  if (narrative && !*narrative) {
```

Comment:
`"narrative":false` will still work to turn stuff off, we just dont reference it anymore from the docs